grammar.js

```diff
@@ -71,7 +70,7 @@ module.exports = grammar({
     // Only conflicts in switch expressions
     [$.lambda_expression, $.primary_expression],
     [$.inferred_parameters, $.primary_expression],
-    [$.class_literal, $.field_access],
+    [$.argument_list, $.record_pattern_body],
```

Comment:
would be great when there's a new conflict to give some intuition about why we have a new one. I know
we usually don't, but I think we should. For example here an example
of ambiguity between an argument and a record_pattern would help.

Reply:
Imo deleting it and running generate is clear enough usually, adding comments to all the conflicts just pollutes the file in my eyes, unless it's a very complex conflict (in this case it's for empty arguments, `case A()`)